beater/auth/anonymous.go

```diff
@@ -15,19 +15,16 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package authorization
+package auth
 
 import (
 	"context"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
 )
 
-func TestDenyAuth(t *testing.T) {
-	handler := denyAuth{}
+// AnonymousAuth implements the Authorizer interface.
+type AnonymousAuth struct{}
```

Comment:
this is not used right now, correct?

Reply:
It's used in the auth middleware when auth isn't required, and in the Jaeger method authenticator for agent config.